orion/primitives/adapters/ncps.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+import tensorflow as tf
+from mlstars.adapters.keras import Sequential
+from mlstars.utils import import_object
+
+
+def build_layer(layer, hyperparameters):
+    layer_class = import_object(layer['class'])
+    layer_kwargs = layer['parameters'].copy()
+    if issubclass(layer_class, tf.keras.layers.Wrapper):
+        layer_kwargs['layer'] = build_layer(layer_kwargs['layer'], hyperparameters)
+    elif issubclass(layer_class, tf.keras.layers.RNN) and isinstance(layer_kwargs['units'], dict):
```

Comment:
Why is RNN checked specifically? Is `test_build_layer_ltc` supposed to cover this branch?

Reply:
Yes it is. The `ncps.tf.LTC` and `ncps.tf.CfC` layers in the `ncps` package are subclasses from RNN.